src/_help/hubs/hub-settings.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Hub settings
+---
+
+- TOC
+{:toc}
+
+## Slug & Custom domains
+
+The slug directly affects the URL of your hubs and documentation. By default, it takes the name of your hub, but modifying it allows you to customize the URL.
+
+Bump.sh does not automatically redirect to the new URL after changing the slug, so be sure to notify your users of this change.
+
+The Custom Domain option allows you to keep your users within your brand experience by fully customizing the URL of your documentation.
+
+> [Custom domain requires additional configuration on your end](/help/custom-domains)
+{: .warning}
+
+## Description
+
+This option allows you to add a description to your hub. For instance, you can explain what this collection includes in terms of documentation, share important details, or even provide a contact for questions.
```

Comment:
Let's also add a mention to the fact this field supports markdown syntax so users can format there text here :)

Reply:
:+1: